microsoft-edge/devtools-guide-chromium/memory-problems/heap-snapshots.md

```diff
@@ -263,47 +235,54 @@ To find an object in the collected heap, you can search using **Ctrl+F** and giv
 <!-- ====================================================================== -->
 ## Uncover DOM leaks
 
-The heap profiler has the ability to reflect bidirectional dependencies between browser native objects (DOM nodes, CSS rules) and JavaScript objects.  This helps to discover otherwise invisible leaks happening due to forgotten detached DOM subtrees floating around.
+The heap profiler has the ability to reflect bidirectional dependencies between browser native objects (DOM nodes, CSS rules) and JavaScript objects.  This helps to discover otherwise invisible leaks that happen because of forgotten detached DOM subtrees that remain in memory.
 
-DOM leaks may be bigger than you think.  Consider the following sample.  When is the `#tree` garbage-collected?
+Consider the following code sample.  When is the element with ID `tree` garbage-collected?
 
 ```javascript
+// Getting a reference to the #tree element.
```

Comment:
```suggestion
// Get a reference to the #tree element.
```
GLOBAL: -ing is quite unusual in code comments.  not needed; omit.  some spoken langs allegedly don't have "-ing".